### ISSUE TITLE
Add an overview of Twitter hashtags

### ISSUE THREAD

**Issue by JonasHelming** [EclipseFdn/ecdtools.eclipse.org#186]
It is currently hard to follow ECDT related topics on Twitter, as different authors are using different hashtags for the same topic. I suggest to add a sub page under resources that lists relevant hashtags from the ecosystem (including projects). This allows people to follow topics more easily, but it also provides a documentation for people doing tweets.

**Comment by brianking** [comment]
A separate page seems too much for me, how about a section on the community page?

**Comment by JonasHelming** [comment]
Sounds very good!

**Comment by ericpoirier** [comment]
Currently looking at the Community page and we already have some suggested hashtags in the Social Media section.

Would it be simply a matter of making this more prominant so that users notice it more?

**Comment by JonasHelming** [comment]
Yes, and add more tags for most important projects. If we add more tags, we somehow need to rearrange the layout to fit

**Comment by ericpoirier** [comment]
@brianking we will need to define a new list of hashtags and to make this section more prominent, we could simply create a Twitter section highligting the hashtags. I think I can manage with the layout changes but can you provide me a new list of hashtags? Or should we ask marketing for help on this?

**Comment by JonasHelming** [comment]
For the projects #eclipseglsp, #eclipsetheia, #emfcloud, #cdtcloud, #eclipseche (to be completed) and
one tag for the working group/ top level project, currently there are two tags. We currently have #ecdtools and #clouddevtools.
With the new Twitter char limit I am leaning towards: #eclipseclouddevtools

**Comment by ericpoirier** [comment]
> For the projects #eclipseglsp, #eclipsetheia, #emfcloud, #cdtcloud, #eclipseche (to be completed) and one tag for the working group/ top level project, currently there are two tags. We currently have #ecdtools and #clouddevtools. With the new Twitter char limit I am leaning towards: #eclipseclouddevtools

Let me know if this is the final list of tags and I will add that to the page.

Thanks.

**Comment by croundy** [comment]
Just chatting with Jacob about this and added him to this issue. We will be sure to consistently use the designated project hashtags and from now on will use #eclipseclouddevtools as the primary hashtag for the working group.

**Comment by ericpoirier** [comment]
> Just chatting with Jacob about this and added him to this issue. We will be sure to consistently use the designated project hashtags and from now on will use #eclipseclouddevtools as the primary hashtag for the working group. This has now been added to our internal social media guidelines.

@croundy Are we simply replacing the current list of hashtags on the Community page with "#eclipseclouddevtools". Or should we also make the hashtag(s) more prominent in the content?

Thanks

**Comment by croundy** [comment]
> @croundy Are we simply replacing the current list of hashtags on the Community page with "#eclipseclouddevtools". Or should we also make the hashtag(s) more prominent in the content?

@ericpoirier the plan is that #eclipseclouddevtools will be used on ALL social media posts. Based on the length of the hashtag, it likely won't often be part of the content, but can be if it makes sense. We may continue to use #ecdtools and @CloudDevTools, along with other industry specific hashtags, but should ALWAYS use #EclipseCloudDevTools whenever posting something from the WG.

**Comment by JonasHelming** [comment]
OK, sounds good. @ericpoirier can we replace the following on the hompepage then:

"Use the #ECDTools #CloudDevTools or #EclipseFdn hashtags whenever relevant in your posts" =>
"Always use the #EclipseCloudDevTools whenever relevant in your posts"

And then a new bullet below:

"Additionally, use a project specific hashtag from the following list : #eclipseglsp, #eclipsetheia, #emfcloud, #cdtcloud, #eclipseche"

**Comment by linkfang** [comment]
Hi @JonasHelming , please preview the updated content here:
https://deploy-preview-224--eclipsefdn-ecdtools.netlify.app/resources/community/

**Comment by JonasHelming** [comment]
Looks good, thank you!

**Comment by linkfang** [comment]
NP! The change is now live!
